pkg/network/kubemacpool.go

```diff
@@ -114,6 +114,10 @@ func renderKubeMacPool(conf *cnao.NetworkAddonsConfigSpec, manifestDir string) (
 	data.Data["CertRotateInterval"] = conf.SelfSignConfiguration.CertRotateInterval
 	data.Data["CertOverlapInterval"] = conf.SelfSignConfiguration.CertOverlapInterval
 
+	ciphers, tlsMinVersion := SecurityProfileSpec(conf.TLSSecurityProfile)
+	data.Data["TLSSecurityProfileCiphers"] = ciphers
```

Comment:
Here we convert to the format expected by the template
```suggestion
	data.Data["TLSSecurityProfileCiphers"] = strings.Join(ciphers, ",")
```